README.rst

```diff
@@ -152,6 +152,27 @@ be able to use this via Docker; the Docker container will not be able to
 access any devices connected to the host ports. You will likely see the
 following error during runtime: "RuntimeWarning: U2F Device Not Found".
 
+Feeding password from stdin
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To enhance usability when using third party tools for managing passwords (aka password manager) you can feed data in
+``aws-google-auth`` from ``stdin``.
+
+When receiving data from ``stdin`` ``aws-google-auth`` disables the interactive prompt and uses ``stdin`` data.
+
+All interactive prompt could be feeded from ``stdin``, but before `#82 <https://github.com/cevoaustralia/aws-google-auth/issues/82>`_
+was not possible to feed the ``Google Password:`` prompt.
+
+Example usage:
+::
+    $ password-manager show password | aws-google-auth
```

Comment:
I think something like `echo "my_password" | aws-google-auth ...` would also work. People who use this tool are likely to know they can swap out the `echo` for anything that writes the password to stdout. Just my 2c.

Reply:
> `echo "my_password" |` 

@mide that is what I'd like to avoid. To expert users, I expect that reading from stdin is a consolidated concept, as is password leaks to shell history. I'd like to avoid a sample command that's a sort of "shooting yourself in the foot" for newcomers or people not so fond on avoiding common pitfalls when using the shell.

Anyway I'm ok whit that solution as is surely clear and explicit, if someone else approves.

Reply:
I agree 100% regarding avoiding "shoot yourself in the foot" examples, but at some point, that isn't our job. I've gone back and forth on this issue quite a bit, and I'm okay either way it lands. 

I think there is value in clarity, but there is also value in preventing silly mistakes. I'm fine using a placeholder for a password manager like you've done. I can go either way.

Reply:
I stand down on my feedback. Let's leave as is.